Backend/app/dashboard/Lib/src/Domain/Types/Transaction.hs

```diff
@@ -58,4 +66,12 @@ data Endpoint
   | ExotelAPI Common.ExotelEndpoint
   | IssueAPI Common.IssueEndpoint
   | VolunteerAPI Common.VolunteerEndpoint
+  | RegistrationAPI BAP.RegistrationEndPoint
+  | SearchAPI BAP.RideSearchEndPoint
+  | SelectAPI BAP.RideSelectEndPoint
+  | QuoteAPI BAP.RideQuoteEndPoint
+  | ConfirmAPI BAP.RideConfirmEndPoint
+  | RBooking BAP.RideBookingEndPoint
```

Comment:
Can be `Booking` only

Reply:
@0utkarsh it will create confusion , as we are already using booking

Reply:
I think it is calling the previous one itself, similar to search, confirm etc. There names are also not changed.